src/components/map/FlowText.js

```diff
@@ -64,7 +65,6 @@ FlowText.propTypes = {
 	link: PropTypes.object.isRequired,
 	startElement: PropTypes.object.isRequired,
 	endElement: PropTypes.object.isRequired,
-	mapText: PropTypes.string.isRequired,
```

Comment:
mapText is not being used inside FlowText. And on reading the code it doesnt look like its being used by any child either.